setup.py

```diff
@@ -242,12 +242,29 @@ def build_extensions(self):
             self.copy_file(src, dst)
 
 
+class mypy_type_check(setuptools.Command):
+    description = 'Run MyPy type checker'
+    user_options = []
+
+    def initialize_options(self):
+        pass
+
+    def finalize_options(self):
+        pass
+
+    def run(self):
+        """Run command."""
+        returncode = subprocess.call(["python", "./tools/mypy-onnx.py"])
+        sys.exit(returncode)
```

Comment:
this can be combined with last line as `subprocess.check_call`.

Reply:
Yes, but check_call throws an exception, which then outputs a backtrace, cluttering the type errors listed by mypy. So it's either check_call and try/catch or call and sys.exit.